plugins/HHAnalyzer.cc

```diff
@@ -231,7 +203,8 @@ void HHAnalyzer::analyze(const edm::Event& event, const edm::EventSetup&, const
             // Note that ID and isolation criteria are in both electron and muon loops
             if (!dilep.isOS)
                 continue;
-            if (event.isRealData() && !(leptons[ilep1].hlt_DR_matchedObject < m_hltDRCut && leptons[ilep2].hlt_DR_matchedObject < m_hltDRCut))
+            // Throw event if there is no matched trigger path
+            if (event.isRealData() && leptons[dilep.ilep1].hlt_idx == -1 && leptons[dilep.ilep2].hlt_idx == -1)
```

Comment:
should be equivalent, but your version is probably safer (also wrt the case discussed above) in case I messed something with the logic in `HHAnalyzer::matchOfflineLepton`... fixed to your version!